### PR TITLE
feat: Adding documentation language switcher

### PR DESCRIPTION
**Pull request by tatang26**
In this PR, I added a language selector for buffalo documentation so you can read the documentation in your most confortable language within those available so far.

I decided to use an inline switcher to get better accessibility to the available languages. The code block is into `languageselector.html` file that is implemented into `baseof.html` documentation template below the `header partial`.

To get all languages availables, I looped into `.AllTranslations` [page variable](https://gohugo.io/content-management/multilingual/#list-all-available-languages) and I compared the current page `.Language` value to define the element which will be highlighted.

To remove the extra spaces generated by hugo template code, I used variable declarion with middle dashes`{{- -}}` regarding with the  [documentation](https://gohugo.io/templates/introduction/#whitespace)

### CODE REVIEW

File: layouts/partials/languageselector.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<div class="container md:flex justify-end md:gap-6 bg-gray-100 py-2">
+    <div class="inline-flex justify items-center gap-2">
+    {{- $currentLanguage := .Language -}}
+    {{- range $index, $page := sort .AllTranslations.Reverse -}}
+        {{ if ne $index 0 }}
+        <span class="border-l-2 h-3/4"></span>
+        {{- end -}}
+        {{- $lang := $page.Language -}}
+        {{$active := eq $currentLanguage $lang }}
+        <a href="{{ $page.RelPermalink }}" title="{{ $lang.LanguageName }}" class="{{ if $active }}text-blue-800 font-bold{{ else }}text-slate-300{{ end }}">{{ $lang.LanguageName }}</a>
```
Comment (sio4) [inline layouts/partials/languageselector.html:11]: could it be a pull-down menu style? just wonder while I am dreaming we have much more supported languages :-)

**Review by sio4 — APPROVED** [review]
I left a comment, but it looks good to me.

File: layouts/partials/languageselector.html (hunk 1 above)
Comment (paganotoni) [reply layouts/partials/languageselector.html:11]: I think given we're only going to have 3 to 4 for now we should keep it like this. I'm hoping to work on using flags or something shorter. I think this one makes a lot more visible the languages the user will be able to call.

**Review by paganotoni — APPROVED** [review]
LGTM. Thanks @tatang26

**Comment by sio4** [comment]
By the way, when I switched to `fr`, I found that all side menus are gone. I think the side menu should show the whole structure even though some pages are not translated. Can we fix this?

**Pull request merged**